tracer/build/_build/Build.Utilities.cs

```diff
@@ -91,6 +91,32 @@ partial class Build
             }
         });
 
+    Target RunInstrumentationGenerator => _ => _
+       .Description("Runs the AutoInstrumentation Generator")
+       .Executes(() =>
+       {
+           var autoInstGenProj = Solution.GetProject("Datadog.AutoInstrumentation.Generator");
```

Comment:
If you do exclude it from the solution as suggested above, you can change this to:
```suggestion
           var autoInstGenProj = SourceDirectory / "Datadog.AutoInstrumentation.Generator" / "Datadog.AutoInstrumentation.Generator.csproj";
```